demo/expressServer.js

```diff
@@ -1,18 +1,35 @@
-var express = require('express');
-var http = require('http');
-var serveIndex = require('serve-index');
+
+import {globPatterns} from '../test/modular/NodeTraceurTestRunner.js';
+
+let express = require('express');
+let http = require('http');
+let serveIndex = require('serve-index');
 
 function servePathAtPort(path, port) {
-  var app = express();
-  app.use(express.static(path));   // before directory to allow index.html to work
+  let app = express();
+  // serveIndex must precede static to allow index.html to work
   app.use(serveIndex(path));
-  var server = http.createServer(app);
+  app.use(express.static(path));
+  // Expand the test list based on the file system.
+  app.get('/traceurService/testGlobs', function(req, res) {
+    let patterns = JSON.parse(req.query.patterns);
+    return globPatterns(patterns).then((files) => {
+      let nodeless = [];
+      files.forEach((file) => {
+        if (file.indexOf('/node/') === -1) {
+          nodeless.push(file);
+        }
+      });
+      return res.send(nodeless);
+    });
+  });
+  let server = http.createServer(app);
   server.on('error', function(e) {
-  	console.log('Port ' + port + ' did not work out');
+    console.error('Port ' + port + ' did not work out');
   });
   server.listen.apply(server, [port]);
   console.log('serving ' + path + ' at ' + port);
 }
 
-servePathAtPort(__dirname + '/..', 8099);
-servePathAtPort(__dirname + '/..', 80);
+servePathAtPort(System.dirname(__moduleName) + '/..', 8099);
```

Comment:
This is a Node script, so __dirname should be available. (i.e., since require is available, so is __dirname). Better to use the "standard" version than the Traceur-specific one, I think.